GitUI/UserControls/FileStatusList.cs

```diff
@@ -242,7 +231,8 @@ public int SelectedIndex
         [Browsable(false)]
         public GitItemStatus SelectedItem
         {
-            get { return SelectedItems.FirstOrDefault(); }
+            get => FileStatusListView.LastSelectedItem()?.Tag<GitItemStatus>();
```

Comment:
to keep stylistic consistency between `get` and `set` in this instance, please have `get { ... }`
the same applies to other similar cases